mynewt-core-project.yml

```diff
@@ -21,9 +21,16 @@ project.name: "apache-mynewt-core"
 
 project.repositories:
     - apache-mynewt-nimble
+    - mcuboot
 
 repository.apache-mynewt-nimble:
     type: github
     vers: 0.0.0
     user: apache
     repo: mynewt-nimble
+
+repository.mcuboot:
+    type: github
+    vers: 0.0.0
```

Comment:
I'm not sure if we should use mcuboot master for this,  also nimble yml needs similar change I guess

Reply:
Why not sure?

Reply:
master is floating target and core release would rather depend on specific version than master branch of external project

I think we should use 1-latest for our tests too.

Reply:
To be clear, the rationale of using latest masters on travis-ci also enables to catch possible build failures although it should not. We do not merge broken commits, so this should always build the test programs. No idea why setting on latest release would be any better. Latest release should be used by people's projects, travis-ci should use master.